content/applications/finance/accounting/payments/checks.rst

```diff
@@ -2,128 +2,118 @@
 | Account              | Statement Match   | Debit    | Credit   |
 +======================+===================+==========+==========+
 | Account Receivable   |                   |          | 100.00   |
 +----------------------+-------------------+----------+----------+
-| Undeposited Funds    |                   | 100.00   |          |
+| Outstanding Receipts |                   | 100.00   |          |
 +----------------------+-------------------+----------+----------+
 
-The invoice is marked as paid as soon as you record the check.
-
-Then, once you get the bank statements, you will match this statement
-with the check that is in Undeposited Funds.
+Then, once you receive the bank statements, match this statement with the check of the **Outstanding
+Receipts** account. This produces the following **journal entry**:
 
 +---------------------+-------------------+----------+----------+
 | Account             | Statement Match   | Debit    | Credit   |
 +=====================+===================+==========+==========+
-| Undeposited Funds   | X                 |          | 100.00   |
+| Outstanding Receipts|        X          |          | 100.00   |
 +---------------------+-------------------+----------+----------+
 | Bank                |                   | 100.00   |          |
 +---------------------+-------------------+----------+----------+
 
-
-If you use this approach to manage received checks, you get the list of
-checks that have not been cashed in the **Undeposit Funds** account
-(accessible, for example, from the general ledger).
+If you use this approach to manage received checks, you get the list of checks that have not been
+cashed in the **Outstanding Receipt** account (accessible, for example, from the general ledger).
 
 .. Note::
-    Both methods will produce the same data in your accounting at the
-    end of the process. But, if you have checks that have not been cashed,
-    this one is cleaner because those checks have not been reported yet on
-    your bank account.
+    Both methods will produce the same data in your accounting at the end of the process. But, if
+    you have checks that have not been cashed in, this method will report uncashed checks on the
+    **Outstanding Receipts** account. However, funds will appear in your bank account whether they
+    are reconciled or not, as the bank value is reflected at the moment of the bank statement.
 
-Option 2: One journal entry only
-================================
+Only one journal entry
+======================
 
 Configuration
 -------------
 
-These is nothing to configure if you plan to manage your checks using
-this method.
+If you wish to have only **one** journal entry, go to :menuselection:`Accounting --> Configuration
+--> Journals --> Bank`. Click `Add a line`, select :guilabel:`Manual` as :guilabel:`Payment Method`,
+and enter `Checks` as :guilabel:`Name`. Click the :guilabel:`vertical ellipsis (⋮)` button, tick
```

Comment:
```suggestion
and enter `Checks` as :guilabel:`Name`. Click the ⋮ button, tick
```
I wouldn't put guilabel here as "vertical ellipsis" doesn't appear on screen. Usually I only add the image of the vertical ellipsis.